nephio-ansible-install/README.md

```diff
@@ -38,11 +38,13 @@ all:
     cloud_user: <username that is used to access the VM>
     github_username: <GitHub username>
     github_token: <GitHub personal access token>
-    github_organization: <optional, if a GitHub organization is used this should be filled out with the GitHub organization>
+    github_uri: github.com <replace this if you have a different github domain name in case of organization hosted github>
```

Comment:
This property is added to other other git hub domains as well, mainly in the case of organization specific git hub domains.